diff --git a/dist/lib/ColorCodes.js b/dist/lib/ColorCodes.js
index 8fd5f96b2e102d3b0a27b254f36d3ba787e2677d..c1f515c00cdd29b10d01d98c29660e2e07abd087 100644
--- a/dist/lib/ColorCodes.js
+++ b/dist/lib/ColorCodes.js
@@ -36,8 +36,8 @@ exports.ColorCodes = {
     [constant_1.Color.Yellow]: {
         highlightHex: "f6f657",
         highlightRGB: [246, 246, 87],
-        shadowHex: "ffff67",
-        shadowRGB: [255, 255, 103]
+        shadowHex: "c38822",
+        shadowRGB: [195, 136, 34]
     },
     [constant_1.Color.Black]: {
         highlightHex: "4a565e",
